mlrun/datastore/targets.py

```diff
@@ -999,7 +1000,7 @@ def as_df(
             start_time=start_time,
             end_time=end_time,
             time_column=time_column,
-            additional_filters=additional_filters,
+            additional_filters=revert_list_filters_to_tuple(additional_filters),
```

Comment:
Why is this needed in the target?

Reply:
to_dataframe can call this function after additional filters passed 
json encode-decode.